doc/manual/src/glossary.md

```diff
@@ -216,6 +216,18 @@
 
   [output path]: #gloss-output-path
 
+- [derived path]{#gloss-derived-path}
```

Comment:
I have come to disagree with this part. It only seems wrong because our old habit of confusing *store path* and *store object*. The store object itself is derivable, but the path *itself* already derived: derived by adding `^...` to the end as needed.

Happy to go with another name, just don't want that regresses on disentangling *store paths* and *store object*.

Reply:
How about "Deriving Path"? A "deriving path" is a path for deriving, just as a "divining rod" is a rod for divining.

Reply:
I agree deriving sounds better. Maybe @fricklerhandwerk has thoughts?

Reply:
@fricklerhandwerk said on call that it sounded good to him too.